### PR TITLE
I corrected some translations to make it more meaningful and clear

### PR DESCRIPTION
**Pull request by LocalAccount1**
I also adjust the language to be more suitable for Vietnamese users

### CODE REVIEW

File: src/_locales/vi.config [hunk 1]
```diff
@@ -14,18 +14,16 @@ Bật
 Tắt
 
 @toggle_current_site
-Chuyển đổi trang web này
+Chuyển đổi chế độ sang/tối cho trang web hiện tại
 
 @setup_hotkey_toggle_site
-Thiết lập trang web hiện tại
-Chuyển đổi phím nóng
+Thiết lập phím nóng để chuyển đổi chế độ sáng/tối cho trang web hiện tại
```
Comment (Gusted) [inline src/_locales/vi.config:20]: Please use two lines

File: src/_locales/vi.config [hunk 2]
```diff
@@ -14,18 +14,16 @@ Bật
...
 
 @toggle_extension
-Chuyển đổi tiện ích mở rộng
+Chuyển đổi chế độ bật/tắt cho tiện ích mở rộng
 
 @setup_hotkey_toggle_extension
-Mở rộng cài đặt
-Chuyển đổi phím nóng
+Thiết lập phím nóng để chuyển đổi chế độ sáng/tối cho tiện ích mở rộng
```
Comment (Gusted) [inline src/_locales/vi.config:26]: Please use two lines

File: src/_locales/vi.config [hunk 3]
```diff
@@ -44,15 +42,13 @@ Vĩ độ
 
 @page_protected
 Trang này được bảo vệ
-bằng trình duyệt
+bởi trình duyệt
 
 @local_files_forbidden
-Không được phép truy
-cập Tập tin có sẵn
+Không thể truy cập các tập tin có sẵn
```
Comment (Gusted) [inline src/_locales/vi.config:48]: Please use two lines

File: src/_locales/vi.config [hunk 4]
```diff
@@ -44,15 +42,13 @@ Vĩ độ
...
 
 @page_in_dark_list
-Trang web này là trên toàn cầu
-"Dark List"
+Trang web này được ghi chú trong danh sách trang web đã ở chế độ tối
```
Comment (Gusted) [inline src/_locales/vi.config:51]: Please use two lines

File: src/_locales/vi.config [hunk 5]
```diff
@@ -61,8 +57,7 @@ Sử dụng bảng màu hệ thống
 Kích hoạt khi chế độ tối của hệ thống được bật
 
 @system_dark_mode_chromium_warning
-This option might not work due to
-a bug in your browser
+Tuỳ chọn này có thể không hoạt động do một lỗi trong trình duyệt của bạn
```
Comment (Gusted) [inline src/_locales/vi.config:60]: Please use two lines

**Comment by LocalAccount1** [comment]
I corrected all the errors you mark.
also thank you for reviewing this pr

File: src/_locales/vi.config [hunk 6]
```diff
@@ -14,18 +14,18 @@ Bật
...
+Đặt phím nóng
 
 @toggle_extension
-Chuyển đổi tiện ích mở rộng
+Chuyển đổi chế độ bật/tắt cho tiện ích mở rộng
 
 @setup_hotkey_toggle_extension
-Mở rộng cài đặt
-Chuyển đổi phím nóng
+Đặt phím nóng
```
Comment (Gusted) [inline src/_locales/vi.config:27]:
```suggestion
Đặt phím nóng
```

File: src/_locales/vi.config [hunk 7]
```diff
@@ -61,8 +61,8 @@ Sử dụng bảng màu hệ thống
 Kích hoạt khi chế độ tối của hệ thống được bật
 
 @system_dark_mode_chromium_warning
-This option might not work due to
-a bug in your browser
+Tuỳ chọn này có thể không hoạt động do
```
Comment (Gusted) [inline src/_locales/vi.config:64]:
```suggestion
Tuỳ chọn này có thể không hoạt động do
```

File: src/_locales/vi.config [hunk 8]
```diff
@@ -44,15 +44,15 @@ Vĩ độ
...
 @local_files_forbidden
-Không được phép truy
-cập Tập tin có sẵn
+Không thể truy cập
+các Tập tin có sẵn
 
 @page_in_dark_list
-Trang web này là trên toàn cầu
-"Dark List"
+Trang web này được ghi chú
```
Comment (Gusted) [inline src/_locales/vi.config:54]:
```suggestion
Trang web này được ghi chú
```

**Pull request merged**